### PR TITLE
Adds ability to configure group display name and unique id LDAP attri…

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary
Adds ability to configure LDAP attributes for group unique identifier and group display name.

#### Ticket Link
n/a

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Needs to be implemented in mobile (link to PR or User Story)
- [x] Has server changes (https://github.com/mattermost/mattermost-server/pull/9923)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

**Comment by mkraft** [comment]
@wiersgallak I have added the descriptions.

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -1294,6 +1294,38 @@ export default {
...
+                            ),
+                            isHidden: needsUtils.not(needsUtils.hasLicenseFeature('LDAPGroups')),
+                        },
+                        {
+                            type: Constants.SettingsTypes.TYPE_TEXT,
+                            key: 'LdapSettings.GroupIdAttribute',
+                            label: t('admin.ldap.groupIdAttributeTitle'),
+                            label_default: 'Group ID Attribute:',
+                            help_text: t('admin.ldap.groupIdAttributeDesc'),
+                            help_text_markdown: true,
```
Comment (jespino) [inline components/admin_console/admin_definition.jsx]: markdown not needed

File: components/admin_console/admin_definition.jsx [hunk 2]
```diff
@@ -1294,6 +1294,38 @@ export default {
                             ),
                             isHidden: needsUtils.not(needsUtils.hasLicenseFeature('LDAPGroups')),
                         },
+                        {
+                            type: Constants.SettingsTypes.TYPE_TEXT,
+                            key: 'LdapSettings.GroupDisplayNameAttribute',
+                            label: t('admin.ldap.groupDisplayNameAttributeTitle'),
+                            label_default: 'Group Display Name Attribute:',
+                            help_text: t('admin.ldap.groupDisplayNameAttributeDesc'),
+                            help_text_markdown: true,
```
Comment (jespino) [inline components/admin_console/admin_definition.jsx]: Markdown not needed

Comment (mkraft) [reply components/admin_console/admin_definition.jsx]: Good point.

**Pull request merged**